developersample/src/main/java/com/microsoft/identity/client/developersample/GraphDataFragment.java

```diff
@@ -145,11 +150,11 @@ public void onDetach() {
         void onExtraScopeRequested();
     }
 
-    private Map convertJsonToMap(final String jsonBlob) {
+    private HashMap convertJsonToMap(final String jsonBlob) {
```

Comment:
Why changing to HashMap, i mean you should always specify type. Return value should be Map<String, String>